src/legacy/Routes.tsx

```diff
@@ -622,10 +622,6 @@ class RouteApp extends React.Component<any, any> {
                         PastelDBThread(rpcConfig)
                       }, period)
                     }
-
-                    this.exportSqliteDBIntervalID = window.setInterval(() => {
-                      saveSqliteDB()
-                    }, exportPastelDBPeriod)
```

Comment:
Are you sure it's safe to remove this?

Reply:
It just has been removed when merging to master branch. will revert them.

Reply:
fixed